### DISCUSSION TITLE
Fadcam_v1.1.2 Stable

### DISCUSSION THREAD

**Discussion by anonfaded** [anonfaded/FadCam#20] [General]
### What's new
- Optimization and bug fixes related to android 13 and above; now app won't crash at startup #19 
- Reduced apk size #18 

| APK File                         | Description                          |
|----------------------------------|--------------------------------------|
| `FadCam_v1.1.2_universal.apk`    | Compatible with all devices (common) |
| `FadCam_v1.1.2_arm64-v8a.apk`    | For 64-bit ARM devices (common)      |
| `FadCam_v1.1.2_armeabi-v7a.apk`  | For 32-bit ARM devices (less common) |
| `FadCam_v1.1.2_x86_64.apk`       | For 64-bit Intel devices (rare)      |
| `FadCam_v1.1.2_x86.apk`          | For 32-bit Intel devices (rare)      |

 **📝Note:** If you're unsure, download the **universal** APK.

<hr /><em>This discussion was created from the release <a href='https://github.com/anonfaded/FadCam/releases/tag/v1.1.2'>Fadcam_v1.1.2 Stable</a>.</em>

**Comment by UdayThyu85123** [comment]
once recording started there should be some notification in notification area. REASON IS... after starting recording, many times we forget to stop it. And it consumes battery and storage then. Like some other apps, we should be able to customize icon and message in notification, so that others cannot know, its recording. For Eg. in one such app, I customize icon of DATA, and in message I write, NEWS UPDATING... So I come to know, its recording.

**Reply by anonfaded** [reply]
Good suggestions. Can you please make a 'feature request' issue for this so we can have it on roadmap?

Also what android version you are using?
On Android 11 a default notification already appears like this:
![Screenshot_20240806-143049_1.jpg](https://github.com/user-attachments/assets/1b3317dd-0c1e-42ac-aadd-9a86c6c0f109)

That may need some fixes to work on newer android versions if it's an issue, confirm that so i can check. Also when recording, do you see any highlights in notification bar? I checked on android 13 emulator and there it showed a green icon at top right area when notification panel was opened, indicating that 'camera and microphone is being used by fadcam' so if its showing for you then isn't it enough? Or having a custom notification is a must? Let me know, and thanks for using FadCam :)

**Comment by UdayThyu85123** [comment]
Hi,

I am using Android version 14, POCO M4 5G mobile. Now, I noticed GREEN DOT
is coming on TOP RIGHT side. That's enough. 😊

For me, if you add clear customized notification, then its good, else this
GREEN DOT is also fine.

Thank you for prompt reply. 😊

**Reply by anonfaded** [reply]
Your request has been added to the project's roadmap
https://github.com/anonfaded/FadCam#upcoming-features

**Comment by UdayThyu85123** [comment]
Today I recorded 2 videos.

1st 18 min- All fine

2nd 39 min - For 3 to 4 min, all fine. After that in while video.... video was STILL, only audio recorded. 

FADCAM version is 1.1.3

**Reply by anonfaded** [reply]
> Today I recorded 2 videos. 1st 18 min- All fine 2nd 39 min - For 3 to 4 min, all fine. After that in while video.... video was STILL, only audio recorded.  FADCAM version is 1.1.3

I got same feedback from 2 other users, and working on it. Will let you know when its fixed. 
See this comment for more details: https://github.com/anonfaded/FadCam/pull/27#issuecomment-2288970813

I think the issue is with the latest android versions where it dont show the persistent notification. I tried it on the android version 11 by recording for 45 minutes and it was all fine, also the notification also just show up onandroid 11 and below, and as your android version is 14 so the issue seems to be bcz of the no persistent notification feature.

**Reply by UdayThyu85123** [reply]
Shall I switch back to 1.1.2 or older version?

**Reply by anonfaded** [reply]
> Shall I switch back to 1.1.2 or older version? 

That won't fix the problem because the issue is with the missing persistent notification during recording. And right now this notification is only shown on android 11 and below. So I'll fix this ASAP, until then you can still use the v1.1.3 but try to not close the app for too long.

Or if you have secondary phone with android version 11 or below, then you can record as long as you like. 

The latest Android versions are too restricted so we are slowly discovering the root causes. Thanks for your patience.

**Reply by UdayThyu85123** [reply]
Thank you for prompt reply.
